schema.sql

```diff
@@ -270,4 +270,9 @@ INNER JOIN species ON animals.species_id = species.id
 WHERE visits.vet_id IN (SELECT id FROM vets WHERE name = 'Maisy Smith')
 GROUP BY species.name
 ORDER BY num_visits DESC
-LIMIT 1;
```

Comment:
- [ ] [OPTIONAL] it would be better to move these queries to `queries.sql` you should only keep tables in the `schema.sql file `
<img width="734" alt="Screenshot 2023-05-09 at 19 13 05" src="https://github.com/Mutai-Gilbert/Vet-Clinic-Database/assets/61790587/9fa00d08-6c6a-4a60-b4ab-0a68bdc8814f">